Dockerfile

```diff
@@ -22,5 +22,7 @@ FROM gcr.io/distroless/base
 WORKDIR /
 ENV DISTRO="debian"
 ENV GOARCH="amd64"
+ENV SERVICE_ADDR="meshery-kuma"
+ENV MESHERY_SERVER="http://meshery:9081"
```

Comment:
Please add: `COPY templates/ ./templates`

Reply:
Yeah added it too✔️